source/core/Timestamp.cpp

```diff
@@ -100,7 +100,7 @@ namespace Vireo
         if (retval == -1) {
             *t = Timestamp(0, 0);
         } else {
-            *t = Timestamp((Double)tv.tv_sec + kStdDT1970re1904, tv.tv_usec / 1E6);
+            *t = Timestamp((((tv.tv_sec * 1E6) + (kStdDT1970re1904 * 1E6)) + tv.tv_usec) / (Double)1E6);
```

Comment:
(Double)1E6 : typecast is redundant. It is already a double.

"Timestamp(tv.tv_sec + kStdDT1970re1904  + (tv.tv_usec / 1E6)) " is less verbose and sufficient.

Reply:
Agree.